src/deps.ts

```diff
@@ -1 +1,2 @@
 export * from "https://deno.land/x/cliffy@v0.25.6/mod.ts";
+export * from "https://deno.land/std@0.170.0/fs/mod.ts";
```

Comment:
suggestion: try the following, if it works - remove direct usage from the `config.command.ts`

```suggestion
export * from "https://deno.land/std@0.170.0/fs/mod.ts";
export * as dir from "https://deno.land/x/dir@1.5.1/mod.ts";
```